### ISSUE TITLE
fix: Custom usernames are possible

### ISSUE THREAD

**Issue by ghost** [githubmonkey/sliderpuzzle_flutter#1]
**Description**
It's possible to use a custom username by utilizing the browser dev tools.

**Steps To Reproduce**

1. Open Dev Tools
2. Go to Application
3. Go to Local Storage
4. Edit value of `flutter.__nickname_collection_key__` such that the new custom name replaces the old one

**Expected Behavior**
Detection of tampering, maybe assigning a new random name?

**Screenshots**
![image](https://user-images.githubusercontent.com/96904024/160846267-9c41c9b8-ab3a-44fc-a99f-d51c1b9204e2.png)

**Comment by githubmonkey** [comment]
Good catch, thanks for reporting! I have plans to move all data to firebase after the competition anyway, which will put a stop to this hack.
